readthedocs/proxito/urls.py

```diff
@@ -78,6 +82,12 @@
         name='project_download_media',
     ),
 
+    # Serve proxied API
+    url(
+        r'^api/v2/',
```

Comment:
Doesn't it need to live under the `DOC_PATH_PREFIX`? We don't want to serve it directly at `docs.example.com/api/`

Reply:
Forgot to add it D: